src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/indentation/commentsindentation/Example3.java

```diff
@@ -0,0 +1,22 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="CommentsIndentation"/>
+  </module>
+</module>
+*/
+package com.puppycrawl.tools.checkstyle.checks.indentation.commentsindentation;
+
+// xdoc section -- start
+public class Example3 {
+  /////////////////////////////// it is OK
+
+  public void foo1() {
+    int a = 0;
+  }
+  // violation below, 'Comment has incorrect indentation level 4, expected is 2'
```

Comment:
Please move it above `foo2()` and make it `violation 2 lines above `

Reply:
Done